app/src/main/java/com/asfoundation/wallet/analytics/RakamAnalyticsSetup.kt

```diff
@@ -0,0 +1,12 @@
+package com.asfoundation.wallet.analytics
+
+import io.rakam.api.Rakam
+
+class RakamAnalyticsSetup : AnalyticsSetUp {
+
+  private val rakamClient = Rakam.getInstance()
+
+  override fun setDefaultWallet(walletAddress: String) {
```

Comment:
If we are setting the id in the AnalyticsSetUp should this method be called setDefaultWallet? In a different case the id can be something different from the wallet address.